easybuild/easyconfigs/a/ARPACK++/ARPACK++-2018.03.26-foss-2017b.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'CMakeMake'
+
+name = 'ARPACK++'
+version = '2018.03.26'
+commit_id = 'e7facee'
+
+homepage = 'https://github.com/m-reuter/arpackpp'
+description = """Arpackpp is a C++ interface to the ARPACK Fortran package,
+ which implements the implicit restarted Arnoldi method for
+ iteratively solving large-scale sparse eigenvalue problems."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = ['https://github.com/m-reuter/arpackpp/archive']
+sources = ['%s.tar.gz' % commit_id]
```

Comment:
@hajgato Please rename the sources on download:

```python
sources = [{'download_filename': '%s.tar.gz' % commit_id, 'filename': SOURCE_TAR_GZ}]

Reply:
@boegel Done